sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -537,6 +510,45 @@ void setSQLiteOpenHelper(@NonNull SQLiteOpenHelper helper) {
         mSQLiteOpenHelper = helper;
     }
 
+    /**
+     * Set maximum SQLite database size.
+     *
+     * @param maxStorageSizeInBytes Maximum SQLite database size.
+     * @return true if database size was set, otherwise false.
+     */
+    boolean setMaxStorageSize(long maxStorageSizeInBytes) {
+        SQLiteDatabase db = getDatabase();
+        long newMaxSize = db.setMaximumSize(maxStorageSizeInBytes);
+
+        /* SQLite always use the next multiple of 4KB as maximum size. */
+        long expectedMultipleMaxSize = maxStorageSizeInBytes / ALLOWED_SIZE_MULTIPLE * ALLOWED_SIZE_MULTIPLE;
```

Comment:
This is a little confusing, can a simple ceiling function do the job too?

Reply:
Unless there's documentation for `ceil()` that I'm missing, it would still have to be in terms of multiples of 4 pages so something like `long res = ceil(max/ALLOWED_MULTIPLE)*ALLOWED_MULTIPLE` which is essentially the same.  Do you think the `ceil()` version is more understandable?

Reply:
I personally do, bu it isn't a huge deal to me either way.